code/game/objects/machinery/vending/marine_vending.dm

```diff
@@ -103,6 +103,7 @@
 			/obj/item/stack/throwing_knife = -1,
 			/obj/item/weapon/twohanded/spear/tactical = -1,
 			/obj/item/ammo_magazine/standard_smartmachinegun = 4,
+			/obj/item/weapon/claymore/harvester/spear = -1,
```

Comment:
minor suggestion here, perhaps move it next to the harvester sword.

Reply:
harvester comes inside scabbard, thus it is in the clothing vendor.

Reply:
fair enough